src/read.c

```diff
@@ -1251,6 +1251,7 @@ void ReadFuncExpr (
     /* 'function( ... arg )' takes a variable number of arguments              */
     if (narg >= 1 && ! strcmp( "arg", CSTR_STRING( ELM_LIST(nams, narg) ) ) )
       {
+	/* TODO: remove this before the next non-beta release */
```

Comment:
This change seems to be unrelated, so perhaps it should not be in this PR?
It also seems fragile. Perhaps we should have a GAP_BETA `#define`, and then the above could become
```
#if !defined(GAP_BETA)
#error Please revise this code before making a non-beta release
#endif
```